scripts/eslint_rules/lib/check_license_header.js

```diff
@@ -69,6 +69,7 @@ const EXCLUDED_FILES = [
 const META_CODE_PATHS = [
   'entrypoints/rn_inspector',
   'panels/rn_welcome',
+  'core/host/RNPerfMetrics.ts',
```

Comment:
@EdmondChuiHW consider adding `front_end/global_typings/react_native.d.ts` here as well in one of the next PRs.

Reply:
Hmm. Looks like `.d.ts` are excluded somehow. The upstream file here doesn't have license and the linter doesn't complain.

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/9370b35e1979ec6953ff59ed78c8751e4f463750/front_end/global_typings/request_idle_callback.d.ts#L1

Added ours to the list for clarity in https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/18